packages/amplify-cli-npm/package.json

```diff
@@ -32,11 +32,12 @@
   "dependencies": {
     "axios": "^0.26.0",
     "rimraf": "^3.0.2",
-    "tar-stream": "^2.2.0"
+    "tar-stream": "^2.2.0",
+    "@aws-amplify/cli-internal": "10.2.0"
```

Comment:
This is intentionally listed as a devDependency below. Otherwise downloading this from npm will cause all of the dependent packages to be downloaded in addition to the binary.